.github/workflows/push-to-docker-hub.yml

```
@@ -0,0 +1,37 @@
+name: Create and push to Docker Hub
```

Let's add more context to avoid confusion with ruby and nodeJS servers
```suggestion
name: Create and push Java server image to Docker Hub
```
Please rename the file accordingly